config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```

Comment:
Guys i don't know if this is a mistake but, you have no routes

Reply:
> Guys i don't know if this is a mistake but, you have no routes

Yes this is because the project is only in the console, it wasn't required to have any views or roots we just use it on the rails console